packages/containers/Import/Steps/DataStep/styles.js

```diff
@@ -59,6 +59,13 @@ export const StyledSelect = styled(Select)`
   }
 `;
 
+export const StyledListSelect = styled(Select)`
+  ${Select.Option} {
```

Comment:
It seems that I fixed it in release/2.0.0 🤔

Reply:
Maybe in another component?

Reply:
yep, in default
`packages/components/Select/styles.js:62`
// not critical